### DISCUSSION TITLE
API rulesets

### DISCUSSION THREAD

**Discussion by LickABrick** [pocketbase/pocketbase#1636] [Q&A]
What are your thoughts about having the possibility to create API rulesets. I quite often reuse a set of API rules between collections, as your app gets bigger and more collections get created it gets harder to maintain your API rules.

Having the possibility to create a set of API rules and attach these to certain collections makes it possible to only maintain the one ruleset, if this ruleset gets changed all linked collections instantly use the modified rules.

**Comment by ganigeorgiev** [comment]
I'm not sure how this will work since each collection can have different fields.
Eventually we may add "presets" and/or option to copy the rules from another collection but still I think it will be confusing and will not work with every collection.
